python/cac_tripplanner/templates/partials/article-detail.html

```diff
@@ -0,0 +1,19 @@
+<article class="info-article">
+    <header class="info-article-header">
+        <div class="info-article-image-hero"style="background-image: url('{{ article.wide_image.url }}');"></div>
```

Comment:
The wide image at the top of the detail view gets cut off, only showing the left half or so. Setting the height to 200px instead of 400px seems to do the trick.